packages/web-extension/test/messaging/remoteApi.test.ts

```diff
@@ -166,6 +168,12 @@ describe('remoteApi', () => {
     expect((sut.consumer as any).addTwo).toBeUndefined();
   });
 
+  it('shutting down exposed object shuts down underlying messenger', async () => {
```

Comment:
```suggestion
  it('shuts down underlying messenger when the exposed object is shut down ', async () => {
```
**it** does x when y happens

Reply:
Thanks, that's way better :sweat_smile: de004619bf9cc1134582f550b4fb6d1014798ac1